pyhoma/models.py

```diff
@@ -22,13 +22,124 @@
 
 
 def obfuscate_id(id: str | None) -> str:
-    return re.sub(r"\d+-", "****-", str(id))
+    return re.sub(r"(SETUP)?\d+-", "****-", str(id))
 
 
 def obfuscate_email(email: str | None) -> str:
     return re.sub(r"(.).*@.*(.\..*)", r"\1****@****\2", str(email))
 
 
+def mask(input: str | None) -> str:
+    return re.sub(r"[a-zA-Z0-9_.-]*", "*", str(input))
```

Comment:
So if None, it will return **** ?
```suggestion
    if input:
      return re.sub(r"[a-zA-Z0-9_.-]*", "*", str(input))
    return None
```

Reply:
Good one!

Reply:
```
pyhoma/models.py:85: error: Argument "repr" has incompatible type "Callable[[Optional[str]], Optional[str]]"; expected "Union[bool, Callable[[Any], str]]"  [arg-type]
```

Apparently, this won't work. It seems that it will only be called when the value is not None.